src/index.js

```diff
@@ -534,7 +534,7 @@ FlashlsSourceHandler.handleSource = function(source, tech, options) {
 videojs.getTech('Flash').registerSourceHandler(FlashlsSourceHandler, 0);
 
 // Use the flashls-enabled version of the video.js SWF
-videojs.options.flash.swf = 'https://players.brightcove.net/videojs-flashls/video-js.swf';
+videojs.options.flash.swf = 'https://unpkg.com/@brightcove/videojs-flashls-swf/dist/video-js.swf';
```

Comment:
Since the swf should be published with videojs-flashls-source-handler we can use that unpkg url instead. Also we can use `__VERSION__` so it uses the right swf version.

 `https://unpkg.com/@brightcove/videojs-flashls-source-handler@__VERSION__/dist/video-js.swf`

Reply:
It looks like the `/es5/` files don't have `__VERSION__` replaced since this just happens with the browserify transform. @gkatsev how do other plugins handle this aside from switching to rollup?